rpgJavaInterpreter-core/src/main/kotlin/com/smeup/rpgparser/interpreter/values.kt

```diff
@@ -454,6 +457,8 @@ data class BooleanValue private constructor(val value: Boolean) : Value {
             is BooleanValue -> value.compareTo(other.value)
             else -> super.compareTo(other)
         }
+
+    fun formatNumeric() = if (value) "1" else "0"
```

Comment:
This function could be private. 
My idea is to expose functions only when necessary, so I don't worry about breaking changes.